tests/instruments/ophir/test_ophir_base.py

```
@@ -0,0 +1,336 @@
+
+def test_units():
+    with expected_protocol(OphirBase, [("$SI", "* W")]) as instr:
+        assert instr.units == "W"
+
+
+# Range
+class TestRange:
+    def test_range_getter(self):
+        with expected_protocol(OphirBase, [
+            ("$AR", "* 3 AUTO 30.0mW 3.00mW 300uW 30.0uW 3.00uW 300nW 30.0nW"),
+            ("$RN", "*4")]) as instr:
```

Flake8 (7.3.0)

[error] visually indented line with same indent as next logical line

(E129)